src/wp-includes/nav-menu-template.php

```diff
@@ -204,14 +204,14 @@ function wp_nav_menu( $args = array() ) {
 		if ( $menu_item->menu_item_parent ) {
 			$menu_items_with_children[ $menu_item->menu_item_parent ] = 1;
 		}
+	}
 
-		// Calculate the depth of each menu item with children
-		foreach ( $menu_items_with_children as $menu_item_key => &$menu_item_depth ) {
-			$menu_item_parent = $menu_items_tree[ $menu_item_key ];
-			while ( $menu_item_parent ) {
-				$menu_item_depth  = $menu_item_depth + 1;
-				$menu_item_parent = $menu_items_tree[ $menu_item_parent ];
-			}
+	// Calculate the depth of each menu item with children
```

Comment:
It's old issue but we should fix this here. As per [PHP Documentation Standard](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#5-1-single-line-comments) comment end with a period.

Reply:
Please note that I just took the code that was already in WordPress and rearranged it (fixed nesting). I've added the dot now. Should I do this in SVN patch too?